spec/namespaces.md

```diff
@@ -95,6 +99,32 @@ namespace N1.N2
 ```
 the two namespace declarations above contribute to the same declaration space, in this case declaring two classes with the fully qualified names `N1.N2.A` and `N1.N2.B`. Because the two declarations contribute to the same declaration space, it would have been an error if each contained a declaration of a member with the same name.
 
+A *single_line_namespace_declaration* permits a namespace declaration to be written without the `{ ... }` block.  For example:
+
+```csharp
+namespace Name;
```

Comment:
Can this appear after usings? If not, there's a subtle difference when you want to convert between the two and the IDE should probably check for that. See this [example](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDYB8ACAmARgFgAoHABgAIA5AgblIDsIw4oAHCAYzhrwDpqAZkoBvUpUmUcI/JQDCBMRKmqZ0gCyUAsgAoAlMpKqTUgBqUAHpQC8lACZwAZhACuGGA2OmT8gPaMUH4YcPwA6gho8LqW+nSUAPQJfPxmKqYAvumUWSS5zKwc3LzUAsJGalS0XmqyeAr14t6m6jhaeoZNPiYW1naOLu6e2d3+gcGhEVFwMXGJybSpI1K5qrn5LGycPDRKXVKwCK5cMJQWoutMm0U7pRUHMEcnZ2KXzZRAA===).

Reply:
> Can this appear after usings?

Yes this can.  Will update this example.

Reply:
actually, going to leave this as is. all we really need to explain is what single-line-namespace is equivalent to.  The externs/usings/attributes that precede it are already explained in the compilation-unit section.